### DISCUSSION TITLE
Do I really need xdg-desktop-portal[-gnome] if I don't use flatpak

### DISCUSSION THREAD

**Discussion by brianjmurrell** [flatpak/xdg-desktop-portal#1394] [Questions]
I don't really understand the concepts behind xdg-desktop-portals but have gathered they are used by the sandboxing that flatpak'd software deployment uses.  So do I need `xdg-desktop-portal[-gnome]` if I am not now using nor ever will use flatpak(s)?

It is currently the second highest resident memory consumer on my system only behind _chrome_.  It's CPU use is not trivial either.  It's currently at 42:37.50 out of 18 days of uptime.

So if I don't actually need it if I am not using any flatpak'd software I could sure use that resident memory and CPU usage back.

**Comment by GeorgesStavracas** [comment]
XDG Portals can be used by sandboxed (Flatpak and Snap) and unsandboxed apps as well

**Comment by hfiguiere** [comment]
> It is currently the second highest resident memory consumer on my system only behind _chrome_.

This is https://gitlab.gnome.org/GNOME/xdg-desktop-portal-gnome/-/issues/91 it seems. I must be doing something because I haven't been able to reproduce.

**Comment by andrei-korshikov** [comment]
To my understanding, you _really_ need xdg-desktop-portal if an application requires (has hard dependency on) it.

I can guess that for common things like open/save dialogs most applications these days will just use GTK/Qt directly. For some more crazy stuff—like screen sharing via PipeWire on Wayland—I can't imagine;)

Anyway, if Portals are optional dependency for a given application, there is no way to answer "in general" what application features are implemented via Portals only.
